src/content/blog/gpt2-mini-token-selection-visualization.md

```
@@ -0,0 +1,201 @@
+   1. 「と」        100.0% ██████████████████████████████████████████████████
+   2. 「ろ」          0.0%
+   ...
+```
+
+確率が一点に集中し、ほぼ確定的に最も確率の高い文字が選ばれる。
+
+結果:「吾輩は猫のごとく、吾輩の」
+
+### 高い Temperature（1.2）
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)